addons/im_livechat/static/src/core/web/composer_patch.js

```diff
@@ -4,6 +4,28 @@ import { _t } from "@web/core/l10n/translation";
 import { patch } from "@web/core/utils/patch";
 
 patch(Composer.prototype, {
+    get navigableListProps() {
+        const props = super.navigableListProps;
+        if (!this.hasSuggestions) {
+            return props;
+        }
+        switch (this.suggestion.state.items.type) {
+            case "Chatbot":
+                {
+                    props.options = this.suggestion.state.items.suggestions.map((item) => {
+                        return {
+                            label: item.name,
+                            bot_id: item.id,
+                            classList: "o-mail-Composer-suggestion",
+                        };
+                    });
```

Comment:
:pick: Removing the `return` statement and wrapping the outer object in parentheses would make this code shorter and simpler.